packages/liveblocks-react-ui/src/primitives/AiMessage/index.tsx

```diff
@@ -0,0 +1,169 @@
+import type {
+  AiToolDefinitionRenderProps,
+  AiToolInvocationPart,
+  Json,
+  MessageId,
+} from "@liveblocks/core";
+import { kInternal } from "@liveblocks/core";
+import { useClient } from "@liveblocks/react";
+import { useSignal } from "@liveblocks/react/_private";
+import { Slot } from "@radix-ui/react-slot";
+import {
+  createContext,
+  forwardRef,
+  useCallback,
+  useContext,
+  useMemo,
+} from "react";
+
+import type {
+  AiMessageContentComponents,
+  AiMessageContentProps,
+} from "./types";
+
+const AI_MESSAGE_CONTENT_NAME = "AiMessageContent";
+
+const defaultMessageContentComponents: AiMessageContentComponents = {
+  TextPart: () => "Default TextPart is not implemented yet",
+  ReasoningPart: () => "Default ReasoningPart is not implemented yet",
+  UploadedImagePart: () => "Default UploadedImagePart is not implemented yet",
```

Comment:
@nimeshnayaju Feel free to update these however you see fit. You can just push your changes to this PR if you'd like.